[Linter] Error running Processing (Processing AssertionError: filePath must be a string)

#### System:
- Windows 10
- Processing 3.4
- Atom 1.31.2
- Processing added to PATH env
- Processing Directory `C:\Program Files\Processing 3\processing-3.4`

#### Packages:
- Linter 2.2.0
- Linter-Processing-Java 0.1.2
- Script 3.18.1
- processing-language 0.2.0*
- processing-autocomplete 0.2.1

**However, after installing `linter-processing-java` and saving any .pde file I see the following error message:**
```
[Linter] Error running Processing
See Console for more info. (Open View -> Developer -> Toggle Developer Tools)
```
Toggle Developer Tools shows:
```
C:\Users\UserName\.atom\packages\linter\lib\linter-registry.js:159 [Linter] Error running Processing AssertionError: filePath must be a string
    at validate (C:\Users\UserName\.atom\packages\linter-processing-java\node_modules\sb-exec\lib\helpers.js:80:24)
    at execProxy (C:\Users\UserName\.atom\packages\linter-processing-java\node_modules\sb-exec\lib\index.js:152:39)
    at Object.exec (C:\Users\UserName\.atom\packages\linter-processing-java\node_modules\atom-linter\lib\helpers.js:105:19)
    at file:///C:/Users/UserName/.atom/packages/linter-processing-java/lib/init.js:45:20
    at new Promise (<anonymous>)
    at Object.lint (file:///C:/Users/UserName/.atom/packages/linter-processing-java/lib/init.js:44:14)
    at promises.push.Promise.then._this2.emitter.emit.number (file:///C:/Users/UserName/.atom/packages/linter/lib/linter-registry.js:101:24)
    at new Promise (<anonymous>)
    at _loop (file:///C:/Users/UserName/.atom/packages/linter/lib/linter-registry.js:99:21)
    at LinterRegistry.<anonymous> (file:///C:/Users/UserName/.atom/packages/linter/lib/linter-registry.js:87:40)
    at Generator.next (<anonymous>)
    at LinterRegistry.<anonymous> (file:///C:/Users/UserName/.atom/packages/linter/lib/linter-registry.js:143:4)
    at Generator.next (<anonymous>)
    at step (C:\Users\UserName\.atom\packages\linter\lib\linter-registry.js:5:273)
    at C:\Users\UserName\.atom\packages\linter\lib\linter-registry.js:5:443
    at new Promise (<anonymous>)
    at LinterRegistry.<anonymous> (C:\Users\UserName\.atom\packages\linter\lib\linter-registry.js:5:99)
    at file:///C:/Users/UserName/.atom/packages/linter/lib/main.js:125:30
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\UserName\AppData\Local\atom\app-1.31.2\resources\app\node_modules\event-kit\lib\emitter.js:27:20)
    at Emitter.module.exports.Emitter.emit (C:\Users\UserName\AppData\Local\atom\app-1.31.2\resources\app\node_modules\event-kit\lib\emitter.js:156:34)
    at file:///C:/Users/UserName/.atom/packages/linter/lib/editor-linter.js:26:20
    at later (C:\Users\UserName\.atom\packages\linter\node_modules\sb-debounce\index.js:9:14)
```

> In order to make sure $PATH is set correctly, you should run Atom from the command line!!! (This is important, elsewise it won't find processing-java).

I've tested running Atom from the Window Command Line to see if this solves the issue, however, I still get the above error!

*If I disable `processing-language` I no longer get the above error, however, Atom does not recognise the file as a Processing (.pde) file and I'm not seeing `linter-processing-java` show any code or syntax errors. Does `linter-processing-java` require `processing-language`?

--- a/lib/init.js
+++ b/lib/init.js
@@ -35,7 +35,7 @@ export function provideLinter() {
       const folderPath = filePath.substring(0, sign);
       // console.log(filePath, folderPath);
 
-      const command = atom.config.get('linter-processing.processingExecutablePath');
+      const command = atom.config.get('linter-processing-java.processingExecutablePath');
       const parameters = [
         `--sketch=${folderPath}`,
         '--build',

